decidim-core/app/cells/decidim/diff_cell.rb

```diff
@@ -49,7 +51,7 @@ def diff_data
     #
     # Returns an HTML-safe string.
     def output_unified_diff(data)
-      return unless data
+      return if data[:new_value].blank?
```

Comment:
Maybe this to not crash if data is nil?

```suggestion
      return if data && data&.[:new_value].blank?
```